translations/combat.tex/pl.po

```diff
@@ -220,6 +298,24 @@ msgid ""
 "{Retreat}\\index{Retreat} or spend 1 MP from the Hero that started the "
 "Combat in order to play another Round."
 msgstr ""
+"W ciągu jednej rundy walki \\textbf{możesz użyć tylko jedną Kartę Zaklęcia}.  Efektów "
+"Krótkotrwałych \\includesvg[height=10px]{\\svgs/ongoing.svg} oraz Aktywacji\\includesvg[height=10px]"
+"{\\svgs/activation.svg} można użyć tylko \\textbf{podczas "
+"Aktywacji swojej Jednostki oraz przed jej atakiem}.  Efekty "
+"krótkotrwałe\\index{Ongoing Effects} trwają do końca Walki lub "
+"do momentu ich wykorzystania.\\par Karty z efektem Natychmiastowym \\includesvg[height=10px]{\\svgs/instant."
+"svg}  można zagrać \\textbf{w dowolnym momencie} oprócz momentu w którym rozpatrywane są "
+"rzuty kością i zadane obrażenia, z wyjątkiem kart których opis sugeruje inaczej.  Karty Natychmiastowe "
+"które podnoszą \\includesvg[height=10px]{\\svgs/attack.svg} lub "
+"\\includesvg[height=10px]{\\svgs/defense.svg} Jednostki, takie jak Karty Statystyk "
+"Ataku i Obrony, \\textbf{muszą} być zagrane po deklaracji "
+"ataku, a ich efekt trwa tylko na czas samego ataku.  "
+"\\subsection*{\\hypertarget{Limit Czasowy}{Limit Czasu Walki}}\\index{Limit "
+"Czasowy} Walka przeciwko Neutralnym Jednostkom ma Limit Czasowy który jest równy \\textbf{jednej "
+"Rundzie Walki}.  Jeśli nie wygrasz Walki przed końcem"
+"Rundy Walki, musisz \\hyperlink{Koniec Walki}"
+"{Wycofać się}\\index{wycofywanie się} lub wydać 1 PR bohatera który rozpoczął "
+"Walkę aby móc ją kontynuować."
```

Comment:
imo kluczowa jest tutaj informacja, że 1 PR to 1 dodatkowa runda, tak jak w angielskiej wersji